Documentation/Updating.md

```diff
@@ -1,13 +1,13 @@
-# Updating to the General Availability (GA) Release
+# Updating to the 2.0.0 Release
 
-Between the RC2 and GA releases of the Microsoft Mixed Reality Toolkit, changes were made that may impact existing projects. This document describes those changes and how to update projects to the GA release.
+Between the RC2 and 2.0.0 releases of the Microsoft Mixed Reality Toolkit, changes were made that may impact existing projects. This document describes those changes and how to update projects to the 2.0.0 release.
```

Comment:
Should we tell people here how to upgrade? I.e do they delete the old MRTK directories and then import package?

Reply:
Also adding on to the list in teams, maybe also add "Review your MRTK profiles in inspector"? Per comment below, DefaultRaycastProvider wasn't set

Reply:
In our RC2.1 release notes, we had upgrade instructions, it was one of our learnings to provide upgrade instructions going forward.

Reply:
Here is my suggested upgrade instructions. I took our RC2.1 instructions, followed them, and added instructions that were missing:

1. Close Unity
2. Delete all MixedRealityToolkit folders from RC1: MixedRealityToolkit, MixedRealityToolkit.Examples, MixedRealityToolkit.Providers, MixedRealityToolkit.SDK, MixedRealityToolkit.Services
3. Delete your Library folder
4. Import the new unity packages (Foundation is required, tools, extensions, and examples are optional). Import Foundation first.
5. In your scenes, delete the MixedRealityToolkit and MixedRealityPlaySpace objects
6. MixedRealityToolkit -> Add to Scene and Configure…

Reply:
added